core/arch/arm/plat-sam/matrix.h

```diff
@@ -31,6 +31,44 @@
 
 #include <tee_api_types.h>
 
+#define SECURITY_TYPE_AS	1
+#define SECURITY_TYPE_NS	2
+#define SECURITY_TYPE_PS	3
+
+#define MATRIX_SPSELR_COUNT	3
+#define MATRIX_SLAVE_COUNT	15
+
+#ifdef CFG_PM_ARM32
+	struct matrix_state {
```

Comment:
remove 1 tab, above and below, for `struct matrix_state` definition.

Reply:
Tabs removed.